### ISSUE TITLE
Detects "Linux Mint" but not "LinuxMint"

### ISSUE THREAD

**Issue by fider-apptimia** [nodkz/mongodb-memory-server#403]
## Versions

- NodeJS: 12.18.4
- mongodb-memory-server-*: 6.7.4 and higher
- mongodb: 4.2.5
- system:
Linux Mint 19.3 Tricia
#52~18.04.1-Ubuntu SMP Thu Sep 10 12:50:22 UTC 2020

- package: mongo-memory-server

## What is the Problem?
Error on npm install:
```
Unknown linux distro LinuxMint
Falling back to legacy MongoDB build!
failed to download/install MongoDB binaries. The error: Error: Status Code is 403 (MongoDB's 404)
This means that the requested version-platform combination doesn't exist
```

Is there any option to fix this so same source code can be used by developers on different OS, without need of using dockerized env?

**Btw.** I also tried to configure `MongoMemoryServer` in runtime via constructor but error message was the same as during npm isntall.

**Comment by hasezoey** [comment]
valid, currently detects `Linux Mint` but not `LinuxMint`
-> this was changed to not detect `Peppermint` as Linux Mint

but are you sure this happen from 6.7.4 on? because that got change in 6.6.5

---

will be fixed in 6.10.x

**Comment by hasezoey** [comment]
fda4f72 should fix this, included in 6.9.2

**Comment by fider-apptimia** [comment]
@hasezoey I tested it with v `6.7.4` and v `6.9.0` - both failed.
With your latest update (v. `6.9.2`) working fine.

Thank you!
